server/app/src/test/java/com/google/heartrate/wearos/app/gatt/heartrate/characteristics/HeartRateGattCharacteristicManagerTest.java

```diff
@@ -0,0 +1,159 @@
+package com.google.heartrate.wearos.app.gatt.heartrate.characteristics;
+
+import android.os.Build;
+
+import com.google.heartrate.wearos.app.gatt.CharacteristicsArgumentProvider;
+import com.google.heartrate.wearos.app.gatt.GattException;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static com.google.heartrate.wearos.app.gatt.CharacteristicsArgumentProvider.Range.IN_UINT16;
+import static com.google.heartrate.wearos.app.gatt.CharacteristicsArgumentProvider.Range.IN_UINT8;
+import static com.google.heartrate.wearos.app.gatt.CharacteristicsArgumentProvider.Range.OUT_UINT16;
+import static com.google.heartrate.wearos.app.gatt.TestUtils.assertNotThrows;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(sdk = {Build.VERSION_CODES.O_MR1}, manifest = Config.NONE)
+public class HeartRateGattCharacteristicManagerTest {
+
+    private HeartRateMeasurementCharacteristic characteristic;
+
+    @Before
+    public void setUp() {
+        characteristic = new HeartRateMeasurementCharacteristic();
+    }
+
+    /**
+     * Assent setting values to given Heart Rate Characteristic was successful.
```

Comment:
nit: typo assent -> assert

Reply:
Fixed in [commit](https://github.com/googleinterns/heartrate-bt-wear/pull/1/commits/f81955e6ee20af370c211fd96123abd538b599e1)